testsuite/libra-swarm/src/swarm.rs

```diff
@@ -294,12 +286,11 @@ impl LibraSwarm {
     }
 
     pub fn configure_validator_swarm(
+        libra_node_bin_path: &Path,
         num_nodes: usize,
         config_dir: Option<String>,
         template: Option<NodeConfig>,
     ) -> Result<LibraSwarm> {
-        LibraNode::prepare();
-
```

Comment:
By eliminating prepare, we increase the likelihood that our ephemeral ports are no longer reserved between the time the config has been generated, compilation, and then execution of the nodes. Thoughts?

Reply:
Actually, remove this and pushing to ensure that the binaries are build before calling into the swarm lib should make it less error prone to run into this specific issue